index.js

```diff
@@ -426,18 +426,10 @@ class K8sExecutor extends Executor {
             url: `${this.podsUrl}/${podName}/status`,
             method: 'GET',
             headers: { Authorization: `Bearer ${this.token}` },
-            https: { rejectUnauthorized: false },
-            retry: {
-                statusCodes: [200],
-                limit: this.maxAttempts,
-                calculateDelay: ({ computedValue }) => (computedValue ? this.retryDelay : 0)
-            },
-            hooks: {
-                afterResponse: [this.scheduleStatusRetryStrategy]
-            }
+            https: { rejectUnauthorized: false }
         };
 
-        const resp = await this.breaker.runCommand(statusOptions);
+        const resp = await request(statusOptions);
 
         logger.debug(`Build ${buildId} pod response: ${JSON.stringify(resp.body)}`);
         if (resp.statusCode !== 200) {
```

Comment:
Might be better to do try/catch
request package considers anything non-2xx/3xx an error
https://github.com/sindresorhus/got/blob/main/documentation/8-errors.md#httperror